model/trace/database.yaml

```
@@ -331,6 +331,9 @@ groups:
 
           For commands that switch the database, this SHOULD be set to the target database (even if the command fails).
 
+          In case a specific query explicitly targets a database different from the one used by the connection,
+          the db.namespace should still be the one from the connection.
+
           If instrumentation cannot reliably determine the current database name, it SHOULD NOT set `db.namespace`.
```

Since we're changing from "don't set if you're not sure" to "the best effort", I think we need to update text around it too.

E.g.

```suggestion
          Instrumentations, especially ones that don't support query parsing, have limited ability to determine the `db.namespace` query is performed against. For example, instrumentations may not be able to reliably detect when application switches active database with `USE {db}` query or provides database name inside fully-qualified table name.
          
          Instrumentations SHOULD do the best effort to determine actual `db.namespace` query is performed against and  SHOULD fallback to the database name associated with the connection or provided in the connection string when this information is available.
```